practice-two/src/style/layouts/header.css

```diff
@@ -0,0 +1,143 @@
+.main-header {
+  background-image: url("../../assets/images/background-header.png");
+  background-position: center;
+}
+
+.navbar__logo {
+  color: var(--color-primary);
+  font-size: var(--font-size-2xlg);
+  font-style: var(--font-weight-primary);
+  font-weight: var(--font-style-default);
+  line-height: var(--line-height-normal);
+  letter-spacing: var(--letter-spacing-xlg);
+}
+
+.navbar__container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.navbar {
+  padding: 0 10px;
+  background-color: var(--color-secondary);
+}
+
+.icon-menu {
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.1);
+  }
+}
+
+.navbar__content {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    background-color: transparent;
+    padding: 15px;
+  }
+
+  .icon-menu {
+    display: none;
+  }
+
+  .navbar__content {
+    display: flex;
+    align-items: center;
+    gap: 50px;
+  }
+
+  .item-content {
+    display: flex;
+    justify-content: space-around;
+
+    &:hover {
+      color: var(--color-primary);
+    }
+  }
+
+  .item-content {
+    color: var(--color-base);
+    font-size: var(--font-size-md);
+    font-style: var(--font-style-default);
+    font-weight: var(--font-weight-primary);
+    line-height: var(--line-height-normal);
+    letter-spacing: var(--letter-spacing-normal);
+  }
+
+  .navbar__list {
+    display: flex;
+    list-style: none;
+    gap: 40px;
+  }
+
+  .item-content {
+    color: var(--color-base);
+    text-decoration: none;
+    align-items: center;
+    font-size: var(--font-size-m);
+    font-style: var(--font-style-default);
+    font-weight: var(--font-weight-primary);
+    line-height: var(--line-height-default);
+    letter-spacing: var(--letter-spacing-medium);
+  }
+
+  .navbar__list-icon {
+    display: flex;
+    align-items: center;
+    gap: 25px;
+  }
+
+  .custom-select {
+    width: 80px;
+    height: auto;
+    border-radius: 20px;
+    border: 1px solid var(--color-base);
+    text-align: center;
+    color: var(--color-base);
+    background-color: var(--color-background);
+    appearance: none;
+    padding: 5px 0;
```

Comment:
Should use padding-top: 5px and padding-bottom: 5px. Because you don't use both 4 padding for it